services/gossip/codec/testutil.go

```diff
@@ -0,0 +1,39 @@
+package codec
+
+import (
+	"reflect"
+)
+
+func emptyPayloads(num int) [][]byte {
+	res := [][]byte{}
+	for i := 0; i < num; i++ {
+		res = append(res, []byte{})
+	}
+	return res
+}
+
+func containsNil(obj interface{}) bool {
```

Comment:
this should go to `test/testify_extensions.go` as `RequireDoesNotContainNil` or a better name